src/trackCopiedText.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Tracking text copied by the user.
+ * 
+ * Adds event listener
+ */
+
+ export default (subscribe) => {
+    window.addEventListener("copy",function(e){
+        ppas_processCopyEvent(e);
+    });
+    
+    const ppas_processCopyEvent = function(e){
```

Comment:
do not use prefixes like ppas, whole file has already isolated scope

Reply:
try to always use camelCase notation to be consistent with the rest of the code

Reply:
Sure, this came from the fact that I literally copied the solution that already existed ;)